WDAC-Policy-Wizard/app/WDAC Wizard.csproj

```diff
@@ -202,13 +208,58 @@
     <Content Include="Resources\imgs\windows-logo.png" />
     <Content Include="Resources\imgs\wizard-ico.ico" />
     <Content Include="Resources\policyTemplates\AllowMicrosoft.xml" />
+    <None Update="priconfig.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
     <None Update="Properties\Settings.settings">
       <Generator>SettingsSingleFileGenerator</Generator>
       <LastGenOutput>Settings.Designer.cs</LastGenOutput>
     </None>
     <None Update="Resources\policyTemplates\cipolicy.xsd">
       <SubType>Designer</SubType>
     </None>
+    <None Update="RulesDict.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Templates\AllowMicrosoft - SingleFormat.xml">
```

Comment:
Same comment as above with respect to all of these new Template\*.xml files. Can you update these references to copy from the existing file paths. E.g. 

```xml
<None Update="Resources\policyTemplates\AllowMicrosoft - SingleFormat.xml">
    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    <TargetPath>Templates\AllowMicrosoft - SingleFormat.xml</TargetPath>
  </None>